tutorials/nbconverted/single_cell_usage.py

```diff
@@ -0,0 +1,227 @@
+    sc_output_files=sc_profile_path, compression_options="gzip"
+)
+
+# saving single-cell morphology dataset
+sc_profile.to_csv(sc_profile_path, compression="gzip")
+
+# displaying dataset
+sc_profile.head()
+
+
+# Now that we have created our merged single-cell profile, let's move on to the next step: loading our `platemaps`. 
+# 
+# Platemaps provide us with additional information that is crucial for our analysis. They contain details such as well positions, genotypes, gene names, perturbation types, and more. In other words, platemaps serve as a valuable source of supplementary information for our single-cell morphology profile.
+
+# In[21]:
+
+
+# loading plate map and display it
+platemap_df = pd.read_csv(plate_map)
+platemap_df.head(8)
+
+
+# ## Annotation
+# 
+# In this step of the walkthrough, we will combine the metadata with the merged single-cell morphology dataset. To accomplish this, we will utilize the `annotation` function provided by `pycytominer`.
+# 
+# The `annotation` function takes two inputs: the merged single-cell morphology dataset and its associated plate map. By combining these two datasets, we will generate an annotated_profile that contains enriched information.
+# 
+# More information about the `annotation` function can be found [here](https://pycytominer.readthedocs.io/en/latest/pycytominer.html#module-pycytominer.annotate)
+# 
+
+# In[22]:
+
+
+# annotating merged single-cell profile with metadata
+annotated_df = annotate(
+    profiles=sc_profile,
+    platemap=platemap_df,
+    join_on=["Metadata_well_position", "Image_Metadata_Well"],
+)
+
+# save annotated profile
+annotated_df.to_csv(anno_profile_path, compression="gzip")
+
+# displaying annotated profile
+annotated_df.head()
+
+
+# ## Noramlization Step
+# 
+# The next step is to normalize our dataset using the `normalize` function provided by `pycytominer`.
+# More information regards `pycytominer`'s `normalize` function can be found [here](https://pycytominer.readthedocs.io/en/latest/pycytominer.html#module-pycytominer.normalize)
+# 
+# Normalization is a critical preprocessing step that improves the quality of our dataset. It addresses two key challenges: mitigating the impact of outliers and handling variations in value scales. By normalizing the data, we ensure that our downstream analysis is not heavily influenced by these factors.
+# 
+# Additionally, normalization plays a crucial role in determining feature importance (which is crucial for our last step). By bringing all features to a comparable scale, it enables the identification of important features without biases caused by outliers or widely-scaled values.
+# 
+# To normalize our annotated single-cell morphology profile, we will utilize the normalize function from pycytominer. This function is specifically designed to handle the normalization process for cytometry data. 
+
+# In[23]:
+
+
+# normalize dataset
+normalized_df = normalize(annotated_df)
```

Comment:
please add some more parameters here, even if all you're using are the defaults. (at the very least, please add `method="standardize"`)